tests/plan/import/mod-plan-data/plans.fmf

```diff
@@ -0,0 +1,18 @@
+/must-be-imported-and-modified:
+    summary: Test of correct environment variable expansion of ${VARIABLE}
+    provision:
+        how: local
+    execute:
+        script: test "${VARIABLE}" == "foobar"
+
+# TODO: Delete 'ref' specification once merged to 'main' and enable previous plan
```

Comment:
The test seems to me quite tight to the plan/import feature. Could it stay here?

I need the `ref:` here to have this test passed during PR check. Once this will be merged, I'll create another PR with removing the `ref:`.